src/neondis.cpp

```diff
@@ -0,0 +1,41 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include "disassembler.h"
+
+int main(int argc, char *argv[])
+{
+    if (argc < 2) {
+        fprintf(stderr, "Usage: %s file1.neonx file2.neonx...fileN.neonx\n", argv[0]);
+        exit(1);
+    }
+
+    int a = 1;
+    while (a < argc) {
```

Comment:
No need for a loop here. It doesn't make sense to disassemble more than one file, especially when they are all written to stdout. Remember the unix mantra: "Do one thing, and do it well."